src/plugin/parsers/scene_graph_parser.cpp

```diff
@@ -45,16 +45,26 @@
 
 void MeshAddedCallback( MObject &node, void *client_data )
 {
-	assert( node.apiType() == MFn::Type::kMesh );
+	if (node.apiType() != MFn::Type::kMesh)
+	{
+		LOGC("Trying to add mesh callback, but node type is not of \"kMesh\".");
+	}
+
   	wmr::ScenegraphParser* scenegraph_parser = reinterpret_cast< wmr::ScenegraphParser* >( client_data );
 
 	// Create an attribute changed callback to use in order to wait for the mesh to be ready
 	scenegraph_parser->GetModelParser().SubscribeObject( node );
+
+	LOG("New mesh added: \"{}\".", MFnMesh(node).fullPathName().asChar());
```

Comment:
Since there's already a mesh added log, maybe specify that this is logged in `MeshAddedCallback`?

Reply:
Done